static/js/saved-query.js

```diff
@@ -159,6 +159,7 @@
  class linkCellRenderer {
      // init method gets the details of the cell to be renderer
      init(params) {
+        Cookies.remove('queryMode');
```

Comment:
You should not remove the cookie because if the user returns to the page again, their default choice gets removed.